src/platforms/common/stm32/dfu_f4.c

```diff
@@ -87,8 +112,8 @@ uint32_t dfu_poll_timeout(uint8_t cmd, uint32_t addr, uint16_t blocknum)
 			return sector_erase_time[sector_num];
 	}
 
-	/* Programming 256 word with 100 us(max) per word*/
-	return 26U;
+	/* Programming 256 words (32-bit) with 16 us(typ), 100 us(max) per word */
+	return 16 * 1024 / 4 / 1000;
```

Comment:
Please sprinkle in some `U` here so this is an unsigned division for an unsigned return and we don't tango with UB.